packages/request-logic/test/unit/actions/cancel.test.ts

```diff
@@ -109,7 +109,7 @@ describe('actions/cancel', () => {
         timestamp: 2,
       });
     });
-    it('cannot cancel by payer with state === accepted', async () => {
+    it('can cancel by payer with state === accepted', async () => {
```

Comment:
Can you keep the original test with a request v2.0.3 and keep the test that it still fail?

Reply:
done